sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_file_system_client.py

```diff
@@ -562,10 +562,10 @@ def get_paths(
         The generator will lazily follow the continuation tokens returned by
         the service.
 
-        :param str path:
+        :param Optional[str] path:
```

Comment:
Updated the docstrings here and in async to accurately reflect the types of parameters passed to `get_paths` API.